api/test/unit/controllers/test_search_controller.py

```diff
@@ -714,6 +716,7 @@ def test_post_process_results_recurses_as_needed(
         .body(re.compile('from":0'))
         .times(1)
         .reply(200)
+        .header("x-elastic-product", "Elasticsearch")
```

Comment:
This really is necessary. The client checks for this header and throws an error about "not supporting an unknown product" :sweat_smile:. It's an easy one trick, luckily!